proto/tripcommon.proto

```diff
@@ -376,6 +376,8 @@ message TaggedValue {
     kLayer = 1;
     kPronunciation = 2;
     kBssInfo = 3;
+    kLevel = 4;
```

Comment:
this file got renamed to `common.proto` in a recent pr i merged. do you mind merging master into this branch or do you want me to do it?

Reply:
merged